### PR TITLE
Print some few links at the end of AWS setup and deploy

### PR DESCRIPTION
**Pull request by nbeloglazov**
### Description

1. The links should help deployer verify whether deployment was successful or not. 
2. At the end of setup we remind to update domain CNAME record to point to the freshly created load balancer url where civiform app is running.
3. Move second run of `terraform apply` to azure-specific post-terraform step. It's not needed by AWS and messes up "print final message" code which is done in post-terraform setup. 

### Checklist

- [x] Added the correct label: < feature | bug | dependencies | infrastructure | ignore-for-release >
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: cloud/aws/bin/deploy.py [hunk 1]
```diff
@@ -22,6 +24,14 @@ def main():
     if config_loader.is_test():
         print('Test completed')
 
+    aws_cli = AwsCli(config_loader)
+    fargate_service = f'{config_loader.app_prefix}-{resources.FARGATE_SERVICE}'
+    tasks_url = aws_cli.get_url_of_fargate_tasks(
```
Comment (azizova-leyla) [inline cloud/aws/bin/deploy.py:29]: inline?

File: cloud/aws/templates/aws_oidc/bin/aws_cli.py [hunk 2]
```diff
@@ -53,9 +53,18 @@ def restart_ecs_service(self, cluster: str, service_name: str):
                 f'--service={service_name}', f'--cluster={cluster}'
             ])
 
+    def get_load_balancer_dns(self, name: str) -> str:
```
Comment (azizova-leyla) [inline cloud/aws/templates/aws_oidc/bin/aws_cli.py:56]: we can get this as output variable from terraform https://registry.terraform.io/modules/cn-terraform/ecs-fargate-service/aws/latest?tab=outputs aws_lb_lb_dns_name

See https://www.terraform.io/cli/commands/output

**Review by azizova-leyla — CHANGES_REQUESTED** [review]

File: cloud/aws/bin/deploy.py (hunk 1 above)
Comment (nbeloglazov) [reply cloud/aws/bin/deploy.py]: Done.

File: cloud/aws/templates/aws_oidc/bin/aws_cli.py (hunk 2 above)
Comment (nbeloglazov) [reply cloud/aws/templates/aws_oidc/bin/aws_cli.py:44]: Yes we can. I don't see much benefit though. We already have to use AWS CLI for various stuff so for consistency I think it's better to use it. One benefit of using terraform is that we don't need to rely on LB name which is defined in terraform and hardcoded in python code. It doesn't seem like a that big of a problem though.

File: cloud/azure/templates/azure_saml_ses/bin/setup.py [hunk 3]
```diff
@@ -56,6 +57,8 @@ def setup_log_file(self):
     def post_terraform_setup(self):
         self._get_adfs_user_inputs()
         self._configure_slot_settings()
+        # Why are running terraform again for azure?
```
Comment (azizova-leyla) [inline cloud/azure/templates/azure_saml_ses/bin/setup.py:60]: we're running it because get_adfs_user_inputs updates secret variables.

**Review by azizova-leyla — APPROVED** [review]

**Pull request merged**